src/main/shell/download_distributions.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+unpack_and_delete_archives() {
+  for f in *.zip; do unzip "$f"; done
+  for f in *.tar.gz; do tar -xvzf "$f"; done
+  # for f in *.gz; do gunzip "$f"; done
+  # for f in *tar; do tar -xvf "$f"; done
```

Comment:
Why do we need this comments on line 6 and 7?

Reply:
Good catch!